HW_4/task_2.py

```diff
@@ -0,0 +1,21 @@
+"""
+Задайте список случайных чисел. Выведите список чисел, которые не повторяются в заданном списке.
+
+Ввод: значение типа <list> (либо значения типа <int> – размерность списка)
+Вывод: значение типа <list>
+
+Пример:
+[1, 1, 2, 3, 3, 4, 5, 5, 6, 7, 7, 8, 9, 9]
+[2, 4, 6, 8]
+"""
+import random
+new_list = [str(random.randint(1, 9))
+            for r in range(int(input('Введите размер списка: ')))]
+clean_list = []
+for i in range(len(new_list)):
+    if (new_list).count(new_list[i]) <= 1:
+        clean_list.append(new_list[i])
```

Comment:
Почему большинство из вас так стремится использовать функцию `range` в цикле `for`, когда этого делать cовершенно не нужно. В цикле for должен присутствовать итерируемый объект, но не только функция `range` дает такой объект. Любая коллекция им является.
И еще вопрос по символу `<=`. Разве может быть подсчёт элемента из списка нулевым или отрицательным?
```suggestion
for i in new_list:
    if new_list.count(i) == 1:  #
        clean_list.append(i)
```